### ISSUE TITLE
Link not correct

### ISSUE THREAD

**Issue by procradminator** [MicrosoftDocs/microsoft-365-docs#7524]
The link on the menu to managing Exchange for partners takes you to the generic managing Exchange with MFA page and is not specific to or containing anything relevant to a partner wishing to manage a delegated tenancy.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 215632a5-7e92-8f85-3b3e-76ed25ff69c4
* Version Independent ID: d4e5810d-2552-ba21-615e-bb369c09ee25
* Content: [Manage Microsoft 365 tenants with Windows PowerShell for DAP partners - Microsoft 365 Enterprise](https://docs.microsoft.com/en-us/microsoft-365/enterprise/manage-microsoft-365-tenants-with-windows-powershell-for-delegated-access-permissio?view=o365-worldwide&source=docs)
* Content Source: [microsoft-365/enterprise/manage-microsoft-365-tenants-with-windows-powershell-for-delegated-access-permissio.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/manage-microsoft-365-tenants-with-windows-powershell-for-delegated-access-permissio.md)
* Service: **o365-administration**
* GitHub Login: @kelleyvice-msft
* Microsoft Alias: **kvice**

**Comment by msbemba** [comment]
@procradminator Thank you for your feedback. The menu for Exhange powershell takes you   to --https://docs.microsoft.com/en-us/powershell/exchange/connect-to-exchange-online-powershell?view=exchange-ps&viewFallbackFrom=o365-worldwide#connect-to-exchange-online-powershell-using-modern-authentication-with-or-without-mfa

The section takes about connecting to PS by following the below command

Connect-ExchangeOnline -UserPrincipalName <UPN> [-ShowBanner:$false] [-ExchangeEnvironmentName <Value>] [-DelegatedOrganization <String>] [-PSSessionOption $ProxyOptions

The DelegatedOrganization parameter specifies the customer organization that you want to manage as an authorized Microsoft Partner. 

Hope this helps!

Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and sharing this explanation. @procradminator Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by procradminator** [comment]
@msbemba OK, now you've pointed it out and I've trawled down the page and read through I can see it. 

However, my point remains valid. The page title in the menu is not the same as the page title you are redirected to. This is not intuitive. The menu item gives the impression you are being redirected to a specific page for connecting to a tenant.

If it was my decision, there would be a page explaining the -Tenancy switch and the implications with a link to the generic page for connecting to Exchange. Perhaps include details on connecting to multiple tenants and closing the session as well as any limitations. Alternatively, change the link title and include in brackets (including tenant connections) with the link directly to the section on the target page.

Bottom line - I still don't think it's clear.
